### DISCUSSION TITLE
Cartesian plane line Series with Axis Values in the middle of crossing section

### DISCUSSION THREAD

**Discussion by rajeshytech** [Live-Charts/LiveCharts2#1223] [General]
Hi,

I have a wpf application and I am using LiveChart2 for Cartesian LineSeries , But I am not able to set the X & Y axis values at the Center of
the two Coordinates with Arrow. Please find my below screenshot, that way I need, please help me.

![MicrosoftTeams-image](https://github.com/beto-rodriguez/LiveCharts2/assets/19607597/9cb6cbb0-89d9-4b47-935a-4287827bc377)

Thanks

**Comment by beto-rodriguez** [comment]
That is not supported yet.

**Comment by rajeshytech** [comment]
Thanks for the conformance. Are you planning to support this in upcoming builds?

**Reply by beto-rodriguez** [reply]
Sure, this is an interesting feature, we can track this at #1211
